### PR TITLE
Added option not to verify cert and same progress bar as gowitness

### PR DESCRIPTION
**Pull request by GOAT-FARM3R**
Had issues where certificate expired so added an option to not verify certs. While at it, the lack of progress bar made me question whether the thing was turned on. Took code from gowitness to make a similar progress bar.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -30,6 +36,7 @@ var (
 	concurrentPtr *int
 	useHTTPSPtr   *bool
 	throttlePtr   *bool
+	dontVerifyCertPrt *bool
```
Comment (leonjza) [inline main.go]: Let's make this `gnoreCertificatePtr`.

File: main.go [hunk 2]
```diff
@@ -103,11 +110,15 @@ func (service *service) testHTTPEgress(port int) {
 		panic(err)
 	}
 
+	tr := &http.Transport{
```
Comment (leonjza) [inline main.go]: Rename this one to `transport`.

**Review by leonjza — COMMENTED** [review]
Small changes.

File: main.go (hunk 1 above)
Comment (GOAT-FARM3R) [reply main.go]: done

File: main.go (hunk 2 above)
Comment (GOAT-FARM3R) [reply main.go]: done

**Pull request merged**